cmd/helm/helm.go

```diff
@@ -56,6 +56,11 @@ func debug(format string, v ...interface{}) {
 	}
 }
 
+func warning(format string, v ...interface{}) {
+	format = fmt.Sprintf("WARNING: %s\n", format)
+	log.Printf(format, v...)
```

Comment:
To echo what @hickeyma had to say, I don't think the internal details of the Go code are relevant to the user here. Do we need them?

Reply:
good call. Will fix.